### PR TITLE
[Bug 1308168] non-translated page should have English link to button, banner

### PR DESCRIPTION
**Pull request by safwanrahman**
So if there are ``fallback_reason``, we should show the ``parent document`` locale as choice also. As the ``document`` is replaced by ``parent_document`` from view, we can only use the ``document`` attributes.
@jwhitlock r?

### CODE REVIEW

File: kuma/wiki/jinja2/wiki/document.html [hunk 1]
```diff
@@ -302,9 +302,10 @@
               {% endif %}
               {% if fallback_reason %}
                 <div id="doc-pending-fallback" class="warning">
-                  <p><bdi>{% trans help_link=help_link, locale=settings.LOCALES[request.LANGUAGE_CODE].native %}
+                  <p><bdi>{% trans help_link=help_link, locale=settings.LOCALES[request.LANGUAGE_CODE].native, parent_language=document.language %}
                     Our volunteers haven't translated this article into <bdi>{{ locale }}</bdi> yet.
-                    <a href="{{ help_link }}">Join us and help get the job done!</a>
+                    <a href="{{ help_link }}">Join us and help get the job done!</a><br>
+                    You can also read the article in <a href="{{ doc_abs_url }}"> {{ parent_language }}.</a>
```
Comment (jwhitlock) [inline kuma/wiki/jinja2/wiki/document.html]: Can you adjust the period placement, so it is outside the link?

```You can also read the article in <a href="{{ doc_abs_url }}">{{ parent_language }}</a>.```

**Comment by safwanrahman** [comment]
@jwhitlock Fixed the issue.
Regarding branch name, I will take care of it in future. Thanks

**Comment by jwhitlock** [comment]
Thanks for the PR!

**Comment by jwhitlock** [comment]
Hmm this ended up with a lot more code than I thought it would. I think I was hasty to merge it.

**Comment by safwanrahman** [comment]
@jwhitlock Sorry, my test debugging code also come up in the commit.
I am very much sorry for that

**Comment by jwhitlock** [comment]
😄 we both were too excited about getting this merged.  I've removed the code with pushes to the master branch, will be more careful next time and won't review code from my phone.

**Pull request merged**